app/helpers/findings_helper.rb

```diff
@@ -69,6 +69,7 @@ def show_review_with_conclusion_status_as_abbr review
   end
 
   def show_finding_review_code_with_decription_as_abbr finding
+    # finding.reload if finding.try(:description).nil?
```

Comment:
WLF?

Reply:
tiene un WIP hermoso el PullRequest =P 
Esto lo puse porque en un momento excluí los clob para que me anduviese todo.
Ya arregle el "problema"  de ` ORA-00972: identifier is too long` el group by escapó una comilla media rara y era todo un gran gran string =P

Reply:
Ahh... jeje, me imaginé que era algo raro... pero bue, por si se había pasado.